go.mod

```diff
@@ -1,20 +1,14 @@
 module github.com/spiffe/spiffe-helper
 
+replace github.com/spiffe/spiffe-helper => ./
```

Comment:
This replace statement should be removed.

Reply:
done